autoload/neomake/quickfix.vim

```diff
@@ -0,0 +1,192 @@
+" vim: ts=4 sw=4 et
+scriptencoding utf-8
+
+let s:maker_match_id = 997
+let s:gutter_match_id = 998
+let s:cursor_match_id = 999
```

Comment:
Not sure either.. let's keep it like this then for now.
For signs base ids of 5000 and 7000 are used.